src/components/dialogmote/referat/ReferatInfoColumn.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import Panel from "nav-frontend-paneler";
+import navFarger from "nav-frontend-core";
+import { FlexColumn } from "../../Layout";
+import { ReactElement } from "react";
+import React from "react";
+
+const InfoPanel = styled(Panel)`
+  background-color: ${navFarger.navLysBlaLighten80};
+  border: 1px solid ${navFarger.navLysBlaDarken40};
+  margin-top: 1.9em;
+`;
+
+interface ReferatInfoColumnProps {
+  children?: ReactElement[] | ReactElement;
```

Comment:
Det går an å gjøre det sånn som dette:
`children?: ReactNode;`
Det er visstnok den mest robuste måten å håndtere React-barn som props på, men mindre man bevisst vil ekskludere noen muligheter. https://linguinecode.com/post/pass-react-component-as-prop-with-typescript

Reply:
ReactNode omfavner mer / er mer generisk enn ReactElement, så den funker på flere tilfeller - men kan være lurt i noen tilfeller å ha en mer spesifikk typing dersom man vet hva children skal brukes til.

Reply:
..så slipper man at de som bruker komponenten kan sende inn feil verdi

Reply:
Da er spørsmålet om den vurderingen er gjort nå? Er det noe vi mener er feil verdi som vi vil ekskludere, eller er det bare i tilfelle?

Reply:
Jeg tenker det er greit å bruke ReactElement[] | ReactElement så sikrer vi f.eks. at man ikke kan sende inn bare en `string` men at det wrappes i en eller flere komponenter fra https://design.nav.no/components/typografi

Reply:
Vil det ikke da være mer naturlig å begrense det til å være en typografi-komponent? 🤷‍♂️ Det føles litt lite intuitivt å gjøre koden mer komplisert, men hvis jeg skal bruke komponenten senere, er det ingenting som sier meg hvorfor jeg ikke får sende inn en string 🤔

Reply:
Godt poeng, kan se på om det lar seg gjøre uten at det blir for komplisert.

Reply:
Det er ikke det verste i verden å ha det som det er, hvis noe annet blir for kronglete. Men interessant diskusjon! 👍

Reply:
Ønsker å kunne sende inn f.eks. `<br />` også, så tror jeg lar `ReactElement[] | ReactElement` være for nå.